source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm

```diff
@@ -269,8 +270,10 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
                               attributes:@{NSFontAttributeName : cell.textLabel.font}
                                  context:nil]
             .size;
-
-    return labelStringSize.height + _spacing;
+    
+    CGFloat rowHeight = labelStringSize.height + _spacing;
+    
+    return (rowHeight < minimumRowHeight) ? minimumRowHeight : rowHeight;
```

Comment:
Lets test the flow. Do we have any UT to validate this change ?

Reply:
There are no tests for ACRChoiceSetView currently. I will work to add unit tests for some components in another PR.